### ISSUE TITLE
conservative_impl_trait is stable

### ISSUE THREAD

**Issue by vitiral** [jonhoo/fantoccini#13]
```
warning: this feature has been stable since 1.26.0. Attribute no longer needed
   --> src/lib.rs:189:12
    |
189 | #![feature(conservative_impl_trait)]
```

**Comment by jonhoo** [comment]
1.26.0 has not actually been released yet :)
You will only get that message if you are currently on a relatively recent `nightly`. Once 1.26 lands I'll get rid of that though!

**Comment by jonhoo** [comment]
Yup, I'm well aware, but futures are nonetheless fully compatible with Rust stable. The upcoming changes to Rust futures are still very much in flux, and under debate, so trying to track that at this time is not really something I want to do. I think it'd also make it pretty painful to rely on this library.

**Comment by vitiral** [comment]
Cool, those are all valid concerns. Thanks for thinking about it and sharing your thoughts.

**Comment by jonhoo** [comment]
Now that 1.26 has been release, I've published 0.8.3, which works on stable!

**Comment by vitiral** [comment]
I plan on starting to use this for `artifact` soon. Things have been put on hold for a couple of weeks, so this is very timely :+1:
